tests/failpoints/cases/test_snap.rs

```diff
@@ -231,18 +231,46 @@ fn test_destroy_peer_on_pending_snapshot() {
     let before_handle_normal_3_fp = "before_handle_normal_3";
     fail::cfg(before_handle_normal_3_fp, "pause").unwrap();
 
+    let pending_remove_is_triggered = Arc::new(AtomicBool::new(false));
```

Comment:
If it's a channel, then L250 doesn't need to sleep an arbitrary time.

Reply:
good point.will fix it.